Makefile

```diff
@@ -27,11 +27,12 @@ GOPATH ?= /tmp/go
 	@touch $@
 
 help:
-	# all - runs verify, build and docker_build targets
-	# test - runs go_test target
-	# build	- runs generate, and then go_build targets
-	# generate - generates pkg/client/ files
-	# verify - verifies generated files & scripts
+	# all				- runs verify, build and docker_build targets
+	# test			- runs go_test target
+	# e2e-test	- runs e2e tests
+	# build			- runs generate, and then go_build targets
+	# generate	- generates pkg/client/ files
+	# verify		- verifies generated files & scripts
```

Comment:
Not too sure the formating looks strange, I guess you need convert tabs to spaces in the comments

Reply:
Fixed.